lib/doorkeeper/models/access_token_mixin.rb

```diff
@@ -15,13 +15,14 @@ module AccessTokenMixin
         inverse_of: :access_tokens
       }
       if defined?(ActiveRecord::Base) && ActiveRecord::VERSION::MAJOR >= 5
-        belongs_to_options.merge(optional: true)
+        belongs_to_options.merge!(optional: true)
```

Comment:
@tute seems hound prefers the above format, should I change it back (it was how my original PR had it)?